okteto.yaml

```diff
@@ -16,7 +16,7 @@ build:
 
 deploy:
   - name: Deploy PostgreSQL
-    command: okteto deploy -f postgresql/okteto.yaml
+    command: okteto deploy -f postgresql/okteto.yaml --var DB_SNAPSHOT_NAME=$DB_SNAPSHOT_NAME --var DB_SNAPSHOT_NAMESPACE=$DB_SNAPSHOT_NAMESPACE
```

Comment:
do we feel like it's better to have a second okteto.yaml for the volumes, or do we keep it here? I'm concerned that this might make it look more complicated than it is.

Reply:
@rberrelleza As it is now, it can be used for future divert scenarios. I mean, if you need to deploy the database in different manifests (the whole app + some specific service for divert), it is better to have a separated manifest to have all the logic there, instead of having to copy the helm command in the different manifest.

But, for what we have today, moving it here might be the best. We can always figure out what is bets when we prepare divert samples